main.bundle.js

```diff
@@ -335,22 +339,51 @@
 
 	var calculateRemainingCalories = function calculateRemainingCalories(meal, totalCalories) {
 	  var remainingCalories = meal[1] - totalCalories;
-	  renderRemainingCalories(meal, remainingCalories);
+	  renderRemainingCalories(meal[0], remainingCalories);
 	};
 
 	var renderRemainingCalories = function renderRemainingCalories(meal, remainingCalories) {
-	  $('#' + meal[0] + '-remaining-calories-count').append('<p class=\'total-remaining-calories\'><strong>' + remainingCalories + '</strong></p>');
+	  $('#' + meal + '-remaining-calories-count').append('<p class=\'total-remaining-calories\'><strong>' + remainingCalories + '</strong></p>');
 	  styleRemainingCalorieCount(meal, remainingCalories);
 	};
 
 	var styleRemainingCalorieCount = function styleRemainingCalorieCount(meal, remainingCalories) {
 	  if (remainingCalories < 0) {
-	    $('#' + meal[0] + '-remaining-calories-count').addClass('negative-remainder');
+	    $('#' + meal + '-remaining-calories-count').addClass('negative-remainder');
 	  } else {
-	    $('#' + meal[0] + '-remaining-calories-count').addClass('positive-remainder');
+	    $('#' + meal + '-remaining-calories-count').addClass('positive-remainder');
 	  }
 	};
 
+	var diaryCaloriesConsumed = function diaryCaloriesConsumed() {
+	  var caloriesConsumed = $('.calories-total-amount');
+	  var totalConsumed = 0;
+	  caloriesConsumed.each(function (index) {
+	    totalConsumed += parseInt(caloriesConsumed[index].innerText);
+	  });
+	  return totalConsumed;
+	};
+
+	var renderTotalCaloriesConsumed = function renderTotalCaloriesConsumed() {
+	  var total = diaryCaloriesConsumed();
+	  $('#diary-calories-consumed-count').append('<p class=\'diary-total-calories-consumed\'><strong>' + total + '</strong></p>');
+	};
+
+	var diaryRemainingCalories = function diaryRemainingCalories() {
+	  var remainingCalories = $('p.total-remaining-calories');
+	  var diaryTotalRemaining = 0;
+	  remainingCalories.each(function (index) {
+	    diaryTotalRemaining += parseInt(remainingCalories[index].innerText);
+	  });
+	  return diaryTotalRemaining;
+	};
+
+	var renderDiaryTotalRemainingCalories = function renderDiaryTotalRemainingCalories() {
+	  var totalRemaining = diaryRemainingCalories();
+	  $('#diary-remaining-calories-count').append('<p class=\'diary-total-remaining-calories\'><strong>' + totalRemaining + '</strong></p>');
```

Comment:
We could probably push this html we're appending to the next line for readability.